wrapanapi/systems/virtualcenter.py

```diff
@@ -241,14 +286,166 @@ def _get_resource_pool(self, resource_pool_name=None):
             return self.system.get_obj(vim.ResourcePool, resource_pool_name)
         elif self.system.default_resource_pool is not None:
             return self.system.get_obj(vim.ResourcePool, self.system.default_resource_pool)
-        return self.system.get_obj_list(vim.ResourcePool)[0]
+        else:
+            return self.system.get_obj_list(vim.ResourcePool)[0]
+
+    def _get_cluster_compute_resource(self, cluster_compute_resource_name=None):
```

Comment:
I'd better call it `the_default_virtual_center_cluster_compute_resource_name`. :)
jokes aside, I'd use `resource_name` or just `name`